app/src/main/res/xml/preferences.xml

```diff
@@ -403,46 +411,56 @@
                 android:title="@string/pref_tags_menu_dismiss" />
             <MultiSelectListPreference
                 android:defaultValue="@array/toggleTagsDefault"
-                android:key="pref-toggle-tags-list"
                 android:enabled="false"
+                android:key="pref-toggle-tags-list"
                 android:title="@string/loading" />
         </PreferenceScreen>
     </PreferenceScreen>
     <PreferenceScreen
         android:key="providers"
         android:summary="@string/providers_summary"
         android:title="@string/title_providers">
-        <PreferenceCategory android:title="@string/search_results_options">
-        <fr.neamar.kiss.preference.SwitchPreference
-            android:defaultValue="true"
-            android:key="enable-contacts"
-            android:title="@string/contacts_name" />
-        <fr.neamar.kiss.preference.SwitchPreference
-            android:defaultValue="true"
-            android:key="enable-settings"
-            android:title="@string/settings_name" />
-        <fr.neamar.kiss.preference.SwitchPreference
-            android:defaultValue="true"
-            android:key="enable-shortcuts"
-            android:title="@string/shortcuts_name" />
-        <fr.neamar.kiss.preference.ResetShortcutsPreference
-            android:dialogMessage="@string/regenerate_shortcuts_desc"
-            android:key="reset"
-            android:title="@string/regenerate_shortcuts" />
-        <fr.neamar.kiss.preference.SwitchPreference
-            android:defaultValue="true"
-            android:key="enable-search"
-            android:title="@string/search_provider_toggle" />
-        <fr.neamar.kiss.preference.SwitchPreference
+        <PreferenceCategory
+            android:key="search-providers"
```

Comment:
Where the new list preference is added, it would be nice to have also related "reset preference"
Similar to existing `ResetExcludedFromHistoryAppsPreference` or `ResetExcludedAppsPreference`

Reply:
Agreed, I'll add that